pkgs/by-name/lu/lubelog/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildDotnetModule
+, dotnetCorePackages
+, fetchFromGitHub
+}:
+
+buildDotnetModule {
+  pname = "lubelog";
+  version = "1.0.9";
+
+  src = fetchFromGitHub {
+    owner = "hargata";
+    repo = "lubelog";
+    rev = "331499461a66dcba3dea2414c23bd4de902cd337";
+    hash = "sha256-uqS3Z7eRj3hRUperzCBOMJsAUbSftu06VVPTWJuXlQw=";
+  };
+
+  projectFile = "CarCareTracker.sln";
+  nugetDeps = ./deps.nix; # File generated with `nix-build -A package.passthru.fetch-deps`.
+
+  dotnet-sdk = dotnetCorePackages.sdk_8_0;
+  dotnet-runtime = dotnetCorePackages.aspnetcore_8_0;
+
+  makeWrapperArgs = [
+    "--set DOTNET_CONTENTROOT ${placeholder "out"}/lib/lubelog"
+  ];
+
+  executables = [ "CarCareTracker" ]; # This wraps "$out/lib/$pname/foo" to `$out/bin/foo`.
+
+  packNupkg = true; # This packs the project as "foo-0.1.nupkg" at `$out/share`.
+
+  meta = with lib; {
+    description = "Vehicle Service Records and Maintainence Tracker";
+    longDescription = ''
+      Self-Hosted, Open-Source, Unconventionally-Named Vehicle Maintenance Records and Fuel Mileage Tracker
+    '';
+    homepage = "https://lubelogger.com";
+    changelog = "https://github.com/hargata/lubelog/releases/tag/v1.0.9";
+    license = licenses.mit;
```

Comment:
> LubeLogger by Hargata Softworks is licensed under the MIT License for individual
and personal use. Commercial users and/or corporate entities are required 
to maintain an active subscription in order to continue using LubeLogger. 

I don't know how to include that commercial part, someone else may help

Reply:
I think it should be defined here, but I'd love to hear what more experienced people than me think about it

Reply:
`unfreeRedistributable` seems best option for me

Reply:
I asked in the Nix/NixOS Matrix room and they said `unfree`. I'm gonna use that and then put this license excerpt in `longDescription`